root/static/scripts/common/components/WorkArtists.js

```diff
@@ -14,7 +14,8 @@ import {bracketedText} from '../utility/bracketed';
 
 import ArtistCreditLink from './ArtistCreditLink';
 
-const COLLAPSE_THRESHOLD = 4;
+const TO_SHOW_BEFORE = 4;
+const TO_TRIGGER_COLLAPSE = TO_SHOW_BEFORE + 1;
```

Comment:
Even though it correctly reproduces existing behavior, it should be `+ 2` (as for collapsed areas) because the “Show {n} more” item takes more place in height than 1 normal item (the shortest), so it makes no sense to hide just 1 item.

```suggestion
const TO_TRIGGER_COLLAPSE = TO_SHOW_BEFORE + 2;
```

Reply:
As you prefer :) Changed.